app/views/layouts/_nav_menu.html.erb

```diff
@@ -1,25 +1,29 @@
 <div class="crayons-dropdown left-2 right-2 s:left-auto p-0 crayons-header__menu__dropdown inline-block">
   <% if user_signed_in? %>
-    <div id="user-profile-link-placeholder" class="border-0 border-b-1 border-solid border-base-20 p-1">
-      <a id="first-nav-link" class="crayons-link crayons-link--block">
-        <div>
-          <span class="fw-medium block">...</span>
-          <small class="fs-s color-base-50"></small>
-        </div>
-      </a>
-    </div>
+    <ul class="p-0" id="crayons-header__menu__dropdown__list">
+      <li id="user-profile-link-placeholder" class="border-0 border-b-1 border-solid border-base-20 p-1">
```

Comment:
```suggestion
      <li id="user-profile-link-placeholder" class="border-0 border-b-1 border-solid border-base-20 p-1 mb-1">
```

This will add a spacing that is currently missing.